e2e/images/default_visibility_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+set -x
+
+declare -r image_name='e2e-images-visibility'
+
+cat <<EOF | kubectl apply -f -
+apiVersion: openstack.gopherkube.dev/v1alpha1
+kind: OpenStackImage
+metadata:
+  labels:
+    app.kubernetes.io/name: openstackimage
+  name: ${image_name}
+spec:
+  cloud: osp1
+  name: ${image_name}
+EOF
+
+kubectl wait --timeout=10s --for=jsonpath='{.status.name}'="${image_name}" OpenStackImage "${image_name}"
```

Comment:
This timeout could be higher IMHO. 30s at least. And maybe we can factor out where we configure this.